### DISCUSSION TITLE
rdk_doc/Algorithm_Application/cdev_dnn_api/cdev_dnn_api

### DISCUSSION THREAD

**Discussion by giscus** [D-Robotics/rdk_doc#48] [General]
# rdk_doc/Algorithm_Application/cdev_dnn_api/cdev_dnn_api

模型推理API接口所用的头文件和lib库文件，请在 《交付物说明》的 package/host/hostpackage/xj3_aarch64/dnn文件夹获取。

https://developer.d-robotics.cc/rdk_doc/Algorithm_Application/cdev_dnn_api/cdev_dnn_api

**Comment by WuChao-2024** [comment]
简单一点就用spdev的接口：https://developer.d-robotics.cc/rdk_doc/Basic_Application/multi_media/multi_media_api/cdev_multimedia_api_x3/bpu_api

商业上直接用libDNN接口，这是最底层的接口：https://developer.d-robotics.cc/api/v1/fileData/horizon_xj3_open_explorer_cn_doc/runtime/source/runtime_dev.html
